.eslintrc.json

```diff
@@ -0,0 +1,29 @@
+{
+    "env": {
+        "browser": true,
+        "es2020": true,
+        "mocha": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended"
+    ],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": 2020,
+        "sourceType": "script",
+        "ecmaFeatures": {
+            "impliedStrict": true
+        }
+    },
+    "plugins": [
+        "@typescript-eslint"
+    ],
+    "rules": {
+        "@typescript-eslint/no-explicit-any": "error",
+        "@typescript-eslint/no-var-requires": "off" /** allows require statements outside of imports **/ 
```

Comment:
I don't understand why allowing requires statements. if it is used in several places and that it is abad practice, it would better to modify the code.

Reply:
I am open to discuss that but maybe lets see Brian's opinion on that too.

Reply:
Not all components are available as TS-level imports. We're limited by asciidoc and others in a few places.